language-server/src/dotty/tools/languageserver/worksheet/WorksheetMessages.scala

```diff
@@ -0,0 +1,15 @@
+package dotty.tools.languageserver.worksheet
+
+import java.net.URI
+
+/** The parameter for the `worksheet/exec` request. */
+case class WorksheetExecParams(uri: URI)
```

Comment:
We should add empty secondary constructors that just pass null/0 to every param like `def this() = this(null)` here, otherwise when Gson deserializes from json, it will need to create the class instance using sun.misc.Unsafe which seems to work but could be problematic on Java 9+